### DISCUSSION TITLE
@alaskaairux/icons@4.35.0^ Module parse failed

### DISCUSSION THREAD

**Discussion by cleung0305** [AlaskaAirlines/WC-Generator#540] [Announcements]
Hi Team, I'm Lok from RAIN team. I'm having these build error when trying to update @alaskaairux/icons to 4.35.0^. Could someone assist me on resolving this?
![image](https://github.com/AlaskaAirlines/WC-Generator/assets/11708607/46f92afa-c1e3-4be8-a232-59b53000ed6e)

**Comment by blackfalcon** [comment]
Hi @cleung0305 

The current version of auro icons is v4.40.2, is there a specific reason why you are installing 4.35.0? Are you specifically targeting this icon for UI use or is this error combined with another Auro element that uses this icon? 

The repo builds are showing successful, so I don't have reason to think that there is an error with the project itself. 

Also, can you please tell us more about your development environment? What I see in that error is that you are using Angular with Webpack? The error states that you may need an additional loader that is not currently configured with your Webpack setup. In our docs we reference [SVG React Loader](https://github.com/AlaskaAirlines/Icons?tab=readme-ov-file#svg-react-loader), but I am unsure if that will work for you. I am sorry, we have little experience with Angular + Webpack projects.

**Reply by cleung0305** [reply]
Hi @blackfalcon , thanks for your reply. I first tried with v4.40.2 and it didn't work. So I went down to 4.35.0 which I believe is the version that started defaulting export to ES6. Anyway, the errors are gone after using Angular's new build system browser-esbuild. I appreciate your help! Thanks!
